From a1a71b1e9dd75124ec24c863c0aa8cef1e1405f6 Mon Sep 17 00:00:00 2001
From: Alexandru Costache <alexandru@balena.io>
Date: Thu, 18 Nov 2021 10:30:13 +0100
Subject: [PATCH] scripts: Fix kernel-module-headers build in Honister

This is an older kernel which is now
built with a newer gcc.

See: https://lkml.org/lkml/2020/4/1/1206

Upstream-status: Backport
Signed-off-by: Alexandru Costache <alexandru@balena.io>
---
 scripts/dtc/dtc-lexer.l | 1 -
 1 file changed, 1 deletion(-)

diff --git a/scripts/dtc/dtc-lexer.l b/scripts/dtc/dtc-lexer.l
index 615b7ec6588f..d3694d6cf202 100644
--- a/scripts/dtc/dtc-lexer.l
+++ b/scripts/dtc/dtc-lexer.l
@@ -38,7 +38,6 @@ LINECOMMENT	"//".*\n
 #include "srcpos.h"
 #include "dtc-parser.tab.h"
 
-YYLTYPE yylloc;
 extern bool treesource_error;
 
 /* CAUTION: this will stop working if we ever use yyless() or yyunput() */
-- 
2.17.1

